### PR TITLE
windows_certificate: add support for obtaining pfx certiifcate data from a certificate store

### PR DESCRIPTION
**Pull request by johnmccrae**
Signed-off-by: John McCrae <john.mccrae@progress.com>

### CODE REVIEW

File: lib/chef/resource/windows_certificate.rb [hunk 1]
```diff
@@ -92,7 +121,9 @@ class WindowsCertificate < Chef::Resource
         default: false,
         introduced: "16.8"
 
-      action :create, description: "Creates or updates a certificate" do
+      action :create do
```
Comment (tas50) [inline lib/chef/resource/windows_certificate.rb:124]: This needs to be the way it was before for the documentation to work

**Comment by tas50** [comment]
Is this something we can do with a property that changes the type vs. an action? The action still seems to be fetch here. Adding a new action per cert type seems to break our overall UX in chef.

File: lib/chef/resource/windows_certificate.rb (hunk 1 above)
Comment (johnmccrae) [reply lib/chef/resource/windows_certificate.rb:124]: Weird, I didn't change that, I thought someone else had.. will put that back

**Comment by tas50** [comment]
This was merged in #11405

**Pull request closed without merging**